test/fixtures/browserstack-app/features/support/env.rb

```diff
@@ -5,6 +5,8 @@
 bs_device = 'ANDROID_9'
 app_location = 'app/build/outputs/apk/release/app-release.apk'
 
+$api_key = "12312312312312312312312312312312"
```

Comment:
What is this for?

Reply:
Currently maze generates a random API key that enables you to use it at runtime.  However as the test fixture is pre-built, it'll need a pre-determined API key.